clang/test/CodeGenSYCL/Inputs/sycl.hpp

```diff
@@ -291,12 +291,22 @@ class spec_constant {
 } // namespace experimental
 } // namespace ONEAPI
 
+class kernel_handler {
+  void __init_specialization_constants_buffer(char *specialization_constants_buffer) {}
+};
+
 #define ATTR_SYCL_KERNEL __attribute__((sycl_kernel))
 template <typename KernelName = auto_name, typename KernelType>
 ATTR_SYCL_KERNEL void kernel_single_task(const KernelType &kernelFunc) {
   kernelFunc();
 }
 
+#define ATTR_SYCL_KERNEL __attribute__((sycl_kernel))
```

Comment:
This shouldn't be needed since it is defined a few lines above.

Reply:
Nice catch! I will remove this